docs/ensembl-help/using-ensembl/ensembl-apps/species-selector/species-selector-intro.md

```diff
@@ -11,20 +11,45 @@ status: draft
 
 # Species selector
 
-The Species selector allows you to view and select the species and genome assemblies that you wish to view in the Genome browser and Entity viewer.
+The Species selector allows you to view and select the species genomes and assemblies that you wish to view in the Genome browser and Entity viewer.
 
 ## Selecting a species
 
-In order to start using the Genome browser and Entity viewer, you first need to select a species and genome assembly. There are two main ways to select a species:
+In order to start using the Genome browser and Entity viewer, you need to select a species genome and assembly. 
 
-- Click on the icons representing our 42 most popular species and click the green 'Add' button.
-- Type the name of a species or genome assembly into the search box and select from the auto-complete options in the drop-down menu. If there is more than one genome assembly available for your species of interest, a drop-down menu will appear allowing you to select your genome assembly of interest. Click the green 'Add' button to add the selected species and genome assembly.
+There are two ways to select a species genome:
+
+1.  Select from the 42 most popular species genomes
+
+> Select the icon representing your species genome of interest from the 42 most popular species genomes and select the green 'Add' button.
+ 
+2.  Search for a species genomes
+
+> Type the name of a species genome or assembly into the search box and select a species genome from the auto-complete options in the drop-down menu. 
+
+If there are more than one genome assembly available for your species of interest, a drop-down menu will appear allowing you to select your genome assembly of interest. 
+Use the green 'Add' button to select the species genome.
+
+When a species genome has been selected the species and assembly name will appear at the top of the page.
+
+## Creating a list of Species 
+
+You can add multiple species genomes and create a list of species genomes in the Species selector. 
+
+The list of species genomes between them when in other apps such as the Genome browser or Entity viewer.
+
+
+## The Species homepage  
+
+The Species homepage contains more information about the genes, variants and assembly of a genome, along with the controls to manage a list of species genomes. 
```

Comment:
'more information on the numbers of genes, variants etc for a genome species, details about the assembly itself, along with...'

Reply:
done